core/bentley/src/StringUtils.ts

```diff
@@ -155,5 +155,5 @@ export function utf8ToString(utf8: Uint8Array): string | undefined {
  * @public
  */
 export function base64StringToUint8Array(base64: string): Uint8Array {
-  return new Uint8Array(atob(base64).split("").map((c) => c.charCodeAt(0)));
+  return new Uint8Array(Buffer.from(base64, "base64").toString("base64").split("").map((c) => c.charCodeAt(0)));
```

Comment:
Well yeah. atob goes from base64 string to ASCII string from which we must extract each byte. Buffer.from goes directly from base64 string to bytes, the rest is unnecessary.

Reply:
Yes, it should probably work there. I didn't even realize that Buffer is Node only...

Reply:
Would some build/webpack error have been produced if we tried to use your version of this function in a frontend module, or only a run-time error?

Reply:
I would think babel would polyfill it but I'll have to check on that. Otherwise it'd probably be a runtime error... I'll run the tests and see if we would have even caught this without running a test app.